packages/ERTP/test/unitTests/mathHelpers/test-natMathHelpers.js

```diff
@@ -61,65 +48,235 @@ test('natMathHelpers', t => {
   );
   t.throws(
     () =>
-      coerce(
-        harden({ brand: { getAllegedName: () => 'somename' }, value: 4n }),
+      m.coerce(
+        {
+          brand: Far('otherBrand', {
+            getAllegedName: () => 'somename',
+            isMyIssuer: async () => false,
+            getDisplayInfo: () => ({}),
+          }),
+          value: 4n,
+        },
+        mockBrand,
       ),
     {
-      message: /The brand in the allegedAmount .* in 'coerce' didn't match the amountMath brand/,
+      message: /The brand in the allegedAmount .* in 'coerce' didn't match the specified brand/,
     },
     `coerce can't take the wrong brand`,
   );
   t.throws(
-    () => coerce(3),
-    { message: /The brand in allegedAmount .* is undefined/ },
+    () => m.coerce(3n, mockBrand),
+    {
+      message: `The brand in amount (a bigint) doesn't look like a brand. Did you pass a value rather than an amount?`,
+    },
     `coerce needs a brand`,
   );
+});
 
-  // getValue
-  t.is(getValue(make(4)), 4n);
+test('natMathHelpers coerce no brand', t => {
+  t.throws(
+    // @ts-ignore
+    () => m.coerce(m.make(4n, mockBrand)),
+    {
+      message: "The brand (an undefined) doesn't look like a brand.",
+    },
+    `brand is required in coerce`,
+  );
+});
 
-  // getEmpty
-  t.deepEqual(getEmpty(), make(0), `empty is 0`);
+test('natMathHelpers getValue', t => {
+  t.is(m.getValue(m.make(4n, mockBrand), mockBrand), 4n);
+  t.is(m.getValue(m.make(4, mockBrand), mockBrand), 4n);
+});
 
-  // isEmpty
-  t.assert(isEmpty({ brand: mockBrand, value: 0n }), `isEmpty(0) is true`);
-  t.falsy(isEmpty({ brand: mockBrand, value: 6n }), `isEmpty(6) is false`);
-  t.assert(isEmpty(make(0)), `isEmpty(0) is true`);
-  t.falsy(isEmpty(make(6)), `isEmpty(6) is false`);
+test('natMathHelpers getValue no brand', t => {
   t.throws(
-    () => isEmpty('abc'),
-    { message: /The brand in allegedAmount .* is undefined/ },
+    // @ts-ignore
+    () => m.getValue(m.make(4n, mockBrand)),
+    {
+      message: "The brand (an undefined) doesn't look like a brand.",
+    },
+    `brand is required in getValue`,
```

Comment:
```suggestion
    `amount is required in getValue`,
```

Reply:
No, this is a misunderstanding. The amount is provided. It is the brand that is required and is absent.